apps/scan/backend/schema.sql

```diff
@@ -6,6 +6,9 @@ create table election (
   precinct_selection text,
   is_test_mode boolean not null default true,
   polls_state text not null default "polls_closed_initial",
+  last_polls_transition_type text,
+  last_polls_transition_time integer,
+  last_polls_transition_ballot_count integer,
```

Comment:
There are a few ways to do this. Could split it out to its own table. Could keep track of all transitions in another table and infer the current state from the last transition. This seemed simplest to start.

Reply:
Yeah this seems fine. One thing to note is that now the `polls_state` field is actually duplicating some of the information held in `last_polls_transition_type`. I think it's still fine to store it for clarity's sake, but thought I'd call it out.